### ISSUE TITLE
Logical mismatch for the number of objectives in the Student scenario

### ISSUE THREAD

**Issue by salexgit** [nicupavel/openpanzer#104]
Hi

There are two victory tiles in the Student scenarios, at the BataafsePe... city with factory facilities.

I do not know whether it is a bug or feature, but in the beginning of the scenario these two tiles are shown with the Belgium flag (also on the strategic map). However, number of objectives left do not account for them, and there are Germany factory facilities in there. It makes me think that the game engine construes them as already captured tiles. It does not affect the game play, but is very very confusing...

I do not know what the original intention was. If the it is assumed that these tiles are already captured by Germans, then why do not show the German flag. 

Regards, Alex

**Comment by salexgit** [comment]
In continuation of this discussion, here is the patch that fixes this issue in the Student map. I checked the drstu02.xml file for the Student scenario, and it looks really weird that the hex is owned by Germans with the German facility unit, but the flag is Belgium. The attached patch just changes the flag to German. 

[openpanzer-student-map.patch.zip](https://github.com/nicupavel/openpanzer/files/1410748/openpanzer-student-map.patch.zip)

**Comment by salexgit** [comment]
Here is the updated original .SCN file in which I just changed "flag" of two hexes, which are for some reason had Holland flag while the tiles are owned by Germans.

[drstu02.scn.zip](https://github.com/nicupavel/openpanzer/files/1887261/drstu02.scn.zip)

Here is the screenshot from the OpenPazner scenario, which I regenerated from the .SCN file. Two victory hexes now have German flags, as they are anyway owned by Germans from the beginning of the game.

![student_correct_flags](https://user-images.githubusercontent.com/22852936/38473964-89e0ab4c-3b87-11e8-8b00-83da6750c944.png)

Just for the sake of comparison, I am also attaching a screenshot where same hexes have Holland flag, which is obviously wrong.

![student_wrong_flags](https://user-images.githubusercontent.com/22852936/38473989-b0206c5c-3b87-11e8-82c2-91f2941a7bf3.png)
